internal/handler/impl/host_handler.go

```diff
@@ -13,11 +13,31 @@ import (
 	"gorm.io/gorm"
 )
 
+const (
+	pendoHostConfSuccess = "HostConf"
```

Comment:
Wondering if make sense to set the value to `HostConfSuccess` to be aligned with the next const value; I think it could provide more semantic; what do you think?